backend/typescript/models/school.model.ts

```diff
@@ -0,0 +1,44 @@
+import mongoose, { Schema, Document } from "mongoose";
+
+/**
+ * This document contains information about a single school.
+ */
+export interface School extends Document {
+  /** the unique identifier for the school */
+  id: string;
+  /** the name of the school */
+  name: string;
+  /** the country the school is located in */
+  country: string;
+  /** the city the school is located in */
```

Comment:
did we choose not to include `sub-region`? i see it in the schema

Reply:
good catch! I missed it - fixed now.